examples/chatgpt/gpt_actions_library/gpt_action_sharepoint_doc.ipynb

```diff
@@ -0,0 +1,640 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# GPT Action Library: Sharepoint (Return as Document)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This page provides an instruction & guide for developers building a GPT Action for a specific application. Before you proceed, make sure to first familiarize yourself with the following information: \n",
```

Comment:
probably worth adding here (and in the other one) a link out to the middleware notebook. I know it's referenced later too